backend/apps/mentorship/api/internal/nodes/module.py

```
@@ -221,4 +227,4 @@ class UpdateModuleInput:
     project_id: strawberry.ID
     project_name: str
     started_at: datetime
-    tags: list[str] = strawberry.field(default_factory=list)
+    tags: list[str] = strawberry.field(default_factory=list)
```

Ruff (0.15.5)

[warning] No newline at end of file

Add trailing newline

(W292)